src/lib.rs

```diff
@@ -20,6 +20,9 @@ mod receiver;
 /// This module contains types representing the strength of an address's reference counting, which
 /// influences whether the address will keep the actor alive for as long as it lives.
 pub mod refcount;
+/// This module contains a way to scope a future to the lifetime of an actor, stopping it before it
+/// completes if the actor it is associated with stops too.
+pub mod scoped_task;
```

Comment:
I think this should be private and re-export the extension trait (if we choose to keep it).